admin.go

```diff
@@ -107,34 +110,44 @@ func (admin AdminConfig) newAdminHandler(addr NetworkAddress) adminHandler {
 		mux:            http.NewServeMux(),
 	}
 
+	addMuxRoute := func(pattern string, moduleName string, h http.Handler) {
```

Comment:
Can we rename this to, like, `addMetricsRoute` or something? or like `addRouteWithMetrics`